handlers.c

```diff
@@ -745,17 +745,25 @@ bool handler__lregions(globals_t * vars, char **argv, unsigned argc)
     return true;
 }
 
+static bool do_search ( scan_match_type_t m, globals_t * vars, uservalue_t * pval, char ** argv, unsigned argc );
```

Comment:
There are style issues in your commits. Don't put any spaces between function name and '(', don't put any spaces behind '(', don't put any spaces between '\*' and variable name and don't put any spaces in front of ')'.
I know there are some legacy parts which have that space between '\*' and the variable name but we're gradually fixing that when we change something at these locations.

So this line should look like this:
```C
static bool do_search(scan_match_type_t m, globals_t *vars, uservalue_t *val, char **argv, unsigned argc);
```
But using forward declaration for static functions is cumbersome as two locations have to be changed if something changes. Please use bottom-up to avoid forward declaration.